From a348d830659fffd2cfc42994524783b07e69b4b5 Mon Sep 17 00:00:00 2001
From: Mathieu Lirzin <mthl@gnu.org>
Date: Sun, 8 Jul 2018 23:41:42 +0200
Subject: [PATCH] python: Don't use '\n' in sed substitution

This change fixes automake bug#31222.

On macOS, 'sed' interprets '\n' in the substitution text as the letter 'n'
instead of as a newline.

* lib/am/python.am [?FIRST?] (am__pep3147_tweak): Use a space instead of '\n'.
* NEWS: Update.
---
 NEWS             | 8 ++++++++
 lib/am/python.am | 2 +-
 2 files changed, 9 insertions(+), 1 deletion(-)

--- lib/am/python.am.orig	2018-02-26 20:38:27 UTC
+++ lib/am/python.am
@@ -97,7 +97,7 @@ endif %?INSTALL%
 if %?INSTALL%
 
 ?FIRST?am__pep3147_tweak = \
-?FIRST?  sed -e 's|\.py$$||' -e 's|[^/]*$$|&.*.pyc\n&.*.pyo|'
+?FIRST?  sed -e 's|\.py$$||' -e 's|[^/]*$$|&.*.pyc &.*.pyo|'
 
 .PHONY uninstall-am: uninstall-%DIR%PYTHON
 uninstall-%DIR%PYTHON:
-- 
2.22.0

